integration/spark/src/main/scala/org/apache/spark/sql/execution/command/index/DropIndexCommand.scala

```diff
@@ -183,17 +183,20 @@ private[sql] case class DropIndexCommand(
           parentCarbonTable)
         parentCarbonTable = getRefreshedParentTable(sparkSession, dbName)
         val indexMetadata = parentCarbonTable.getIndexMetadata
+        var hasCgFgIndexes = false
```

Comment:
Can keep old code only i think, as  hasCgFgIndexes is not used in multiple places. Just change hasCgFgIndexes logic.